presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,62 @@
+
+## Web UI
+
+* Add support for OAuth2 authorization in Web UI. ({issue}`5355`)
+* Fix invalid operator stats reporting in stage performance view. ({issue}`6114`)
+
+## JDBC Driver Changes
+
+* Allow reading `timestamp with time zone` value as `ZonedDateTime` using `ResultSet.getObject(int column, Class<?> type)` method. ({issue}`307`)
+* Change representation of a `row` value. `ResultSet.getObject` now returns an instance of `io.prestosql.jdbc.Row` class, which better represents
+  the returned value. Previously a `row` value was represented as a `Map` instance, with unnamed fields being named like `field0`, `field1`, etc. 
+  You can access the previous behavior by invoking `getObject(column, Map.class)` on the `ResultSet` object. ({issue}`4588`)
+* Fix failure when reading a `timestamp` or `timestamp with time zone` value with second fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
+* Fix element representation in arrays returned from `ResultSet.getArray`, making it consistent with `ResultSet.getObject`.
+  Previously the elements were represented using internal client representation (e.g. `String`). ({issue}`6048`)
+* Fix `ResultSetMetaData.getColumnType` for `timestamp with time zone`. Previously the type was miscategorized as `java.sql.Types.TIMESTAMP`. ({issue}`6251`)
+* Fix failure when instance of `SphericalGeography` Geospatial type is returned in the `ResultSet`. ({issue}`6240`)
+
+## CLI Changes
+
+* Fix rendering of `row` values with unnamed fields. Previously they were printed using fake field names like `field0`, `field1`, etc. ({issue}`4587`)
+* Fix query progress reporting. ({issue}`6119`)
+* Fix failure when instance of `SphericalGeography` Geospatial type is returned to the client. ({issue}`6238`)
+
+## Hive Connector Changes
+
+* Reduce scheduling latency for queries where scanned files that are referenced by symlinks
+  are located in same directory. ({issue}`6158`)
+* Add deserializer class name to hive connector split information, which is exposed through the event listener. ({issue}`6006`)
+* Improve parallelism for queries where scanned files are referenced by symlinks. ({issue}`6213`)
+
+## Iceberg Connector Changes
+
+* Optimize predicate pushdown by using filters on non-partition columns during split generation and table scan. ({issue}`4932`)
+
+## Kafka Connector Changes
+
+* Allow writing `timestamp with time zone` values into columns using `milliseconds-since-epoch` or `seconds-since-epoch` JSON encoders. ({issue}`6074`)
+
+## Other Connector Changes
+
+* Fix table metadata caching for PostgreSQL, MySQL, SQL Server, Redshift, MemSQL and Phoenix connectors:
+  * Previously cache did not store information if table exists and it was always verifing that in remote database.  ({issue}`6081`)
+  * Consider session properties when caching metadata, so there are different cache entries for diffent session properties. ({issue}`6167`)
```

Comment:
Can this be described more from user perspective?

This is actually "Fix potential query failures when using caching" situation but the wording doesn't apparently communicate this.